packages/fluentui/react-northstar/src/components/SplitButton/SplitButton.tsx

```diff
@@ -112,16 +115,20 @@ class SplitButton extends AutoControlledComponent<WithAsProp<SplitButtonProps>,
     primary: customPropTypes.every([customPropTypes.disallow(['secondary']), PropTypes.bool]),
     secondary: customPropTypes.every([customPropTypes.disallow(['primary']), PropTypes.bool]),
     toggleButton: customPropTypes.itemShorthand,
+    alignRight: PropTypes.bool,
```

Comment:
This is not scalable 💇‍♂ 

Do you have a screenshot how it should look? Is there a real need to change `target`?

Reply:
It should look like this:

<img width="291" alt="Screen Shot 2020-04-08 at 8 50 57 PM" src="https://user-images.githubusercontent.com/8545105/78822044-b386aa80-79da-11ea-9e9b-86c1a8f8a0a1.png">

About the target, the popup has no idea about the right button on the `<SplitButton />` so wouldn't never be aligned on its end

Reply:
As we spoke, alternatively we can set the container `<ElementType />` as the ref. I guess it can make more sense, but let's decided it....

Reply:
For me it makes sense

Reply:
Solved